### PR TITLE
avt_vimba_camera: 2001.1.0-1 in 'foxy/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by icolwell-as**
Increasing version of package(s) in repository `avt_vimba_camera` to `2001.1.0-1`:

- upstream repository: https://github.com/astuff/avt_vimba_camera.git
- release repository: https://github.com/astuff/avt_vimba_camera-release.git
- distro file: `foxy/distribution.yaml`
- bloom version: `0.10.7`
- previous version for package: `null`

## avt_vimba_camera

```
* Port to ROS2 (#56 <https://github.com/astuff/avt_vimba_camera/issues/56>)
* Update README for ROS2 (#58 <https://github.com/astuff/avt_vimba_camera/issues/58>)
* Contributors: icolwell-as
```

### CODE REVIEW

File: foxy/distribution.yaml
Comment (ivanpauno) [inline foxy/distribution.yaml:371]: could you add a top level license file to the repo?

**Review by ivanpauno — DISMISSED** [review]

File: foxy/distribution.yaml
Comment (icolwell-as) [reply foxy/distribution.yaml:371]: Will do! this was originally forked from https://github.com/srv/avt_vimba_camera, which has no licence file, but the package.xml states "BSD". I'll create a BSD licence file if that sounds like the best approach.

Comment (ivanpauno) [reply foxy/distribution.yaml:371]: Sorry, I didn't notice you were releasing from a fork.
Do you have approval from the upstream maintainers to make a release?

In general, why prefer not allowing releases from forks.

Comment (icolwell-as) [reply foxy/distribution.yaml:371]: We were never able to make contact and assumed it was a dead project. See here: https://github.com/srv/avt_vimba_camera/issues/39

Comment (ivanpauno) [reply foxy/distribution.yaml:371]: I see.

That seems fine to me, I'll wait for an approval of a second reviewer in this case.
Maybe @sloretz or @hidmic (?).

**Review by ivanpauno — APPROVED** [review]

File: foxy/distribution.yaml
Comment (hidmic) [inline foxy/distribution.yaml:368]: @icolwell-as `2001` is somewhat unusual for a major version.

File: foxy/distribution.yaml
Comment (hidmic) [reply foxy/distribution.yaml:371]: I'm not sure we have a policy for these cases. @clalancette @tfoote ?

**Review by hidmic — APPROVED** [review]
Overall LGTM

File: foxy/distribution.yaml
Comment (icolwell-as) [reply foxy/distribution.yaml:368]: See discussion starting from here:
https://discourse.ros.org/t/ros2-transition-strategy/9154/14?u=ian_colwell
It's the best way we could think of for managing both ros1 and ros2 packages coming out of a single git repo.

File: foxy/distribution.yaml
Comment (clalancette) [reply foxy/distribution.yaml:371]: We do have a policy where we encourage people to contact the upstream and get the releases and changes done there.  However, if the original repository maintainers can't be contacted, we can release from a fork.

In this case, it does look like that was done, so :+1: from me on using this fork.

**Review by sloretz — APPROVED** [review]

**Pull request merged**
